get_authors.rb

```diff
@@ -2,8 +2,40 @@
 # This script gets last 5 authors of a file and writes them at the end of the file.
 
 # Get last 5 authors of a file and their email
```

Comment:
This does not seem like a correct comment. Could you, maybe move the one from bellow (L6) here?

Reply:
Thanks for spotting, overlooked this one.